### PR TITLE
fix: Core Carousel navigation buttons - focus/active state in Safari and Firefox browser

### PR DESCRIPTION
**Pull request by mykolavorobiov**
## Related Issue
https://github.com/SAP/fundamental-ngx/issues/3753
## Description
Fixed focus and pressed (active) states for carousel navigation buttons for Safari and Firebox browsers.
### Before:
image
### After:
Focus
image
Active
image

#### Please check whether the PR fulfills the following requirements
2. The code follows fundamental-styles code standards and style
- [x] Mixins are used for repeatable code (`fd-active`, `buttonFocus`)
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11

### CODE REVIEW

**Review by InnaAtanasova — APPROVED** [review]

File: src/carousel.scss [hunk 1]
```diff
@@ -13,6 +14,15 @@ $button: #{$fd-namespace}-button;
   $fd-carousel-dot-size: 0.25rem !default;
   $fd-carousel-dot-active-size: 0.5rem !default;
 
+  @mixin after-position-overwrite() {
+    &::after {
```
Comment (JKMarkowski) [inline src/carousel.scss:18]: You can remove &::after from this mixin and place `@include after-position-overwrite` inside 
`@include buttonFocus() { @include after-position-overwrite();  }`

**Review by JKMarkowski — COMMENTED** [review]
Hi @mykolavorobiov Check my comment and let me know when it's addressed, so I will revisit this PR

**Comment by droshev** [comment]
@mykolavorobiov travis is failing

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Pull request merged**